spec/support/chargeback_helper.rb

```diff
@@ -10,6 +10,18 @@ def set_tier_param_for(metric, param, value, num_of_tier = 0)
       def used_average_for(metric, hours_in_interval, resource)
         resource.metric_rollups.sum(&metric) / hours_in_interval
       end
+
+      def add_metric_rollups_for(resources, range, step, custom_params, trait = :with_data)
```

Comment:
1) Is the `trait` parameter used at all? If everything uses the default, maybe just don't even bother passing it in until it's actually needed.

2) Let's make this method handle singles vs multiples. Do `Array(resources).each do |resource| ...` and update all the callers that pass an array of one resource just to pass the resource. 

3) `custom_params` doesn't have any context; can you rename to something like `metric_rollup_params` or something else that says 'these params are what is passed to create a metric rollup'?